app/controllers/spree/admin/taxjar_backfills_controller.rb

```diff
@@ -2,9 +2,9 @@ module Spree
   module Admin
     class TaxjarBackfillsController < Spree::Admin::BaseController
       def create
-        ::SuperGood::SolidusTaxjar::BackfillTransactions.call
-        flash[:success] = "Transactions Backfilled!"
```

Comment:
I think there was a test for this wasn't there? We probably don't want to be adding this line with a test if we're going to delete it in the same PR.

Reply:
The implementation changed over the course of the PR as the feature developed. Initially, a flash message was sufficient communication, but then as more logs were added, displaying them on the page itself was needed.

This line isn't the final implementation, but it's part of the history of building up the implementation.

Reply:
Hmm, I'm not sure if this is accurate but it seems like things were committed when they were written as opposed to when it would make the most sense for someone reading. i.e. committing an empty service class earlier and committing one behaviour then changing it later. I can see how this might happen but I don't think it's ideal for either the reviewers or anyone looking back at this history later.